### PR TITLE
to make it easier to translate

### PR DESCRIPTION
**Pull request by dyaskur**
If no problem with this, I will do more like this.

Sorry if any wrong/bad practice, feel free to give criticism and suggestions

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class ApplicationController < ActionController::Base
   before_action :enforce_2fa
   before_action :block_write_request, if: :read_only_mode?
 
-  helper_method :top_level_post_types, :second_level_post_types
```
Comment (ArtOfCode-) [inline app/controllers/application_controller.rb:16]: You don't actually need this; helpers are available in views by default. You can remove this and just call `i18ns` directly in the views.

**Review by ArtOfCode- — CHANGES_REQUESTED** [review]
Looks good, with one slight change needed. Making things easier to translate and people who can provide translations are definitely things we need, we just haven't had time to do it ourselves.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (dyaskur) [reply app/controllers/application_controller.rb]: removed, I see I can use `t` method too, so I changed `i18ns` to `t` .
Thanks

**Review by ArtOfCode- — APPROVED** [review]

**Comment by ArtOfCode-** [comment]
Thanks!

**Comment by cellio** [comment]
Thanks @dyaskur!  What should folks know going forward, to avoid regressing?  For example, if we add new UI elements that have text, what's the right way to do it?  (The question on collab.codidact.org about this might be a good place to document best practices for new code.)

**Pull request merged**